resources/views/pages/partials/stream.blade.php

```diff
@@ -43,9 +47,9 @@ class="flex px-2 py-1 space-x-2 bg-gray-900 rounded bg-opacity-80 backdrop-filte
 
         <ul class="flex flex-wrap gap-6">
             <li>
-                <a href="{{ $stream->toWebcalLink() }}"
+                <a href="{{ route('calendar.ics.stream', $stream) }}"
```

Comment:
I think this change was from an old stage. Might be it came in through a conflict? The `toWebcalLink` method should be used.

Reply:
Fixed!